documentation-site/components/gallery.js

```diff
@@ -81,6 +81,14 @@ const COMPONENTS = {
       href: '/components/select',
       Component: thumbnails.SvgSelect,
     },
+    {
```

Comment:
should we position these next to the datepicker?

Reply:
It's sorted alphabetically so that would be strange. Maybe we could create a new "date & time" category?

Reply:
yeah, that's true - I was just thinking, that for me they belong to the same logical thing

maybe a new category would make sense, yeah

Reply:
Everyone fine with "DATE & TIME"? @nadiia @sandgraham @chasestarr

Reply:
sure